lib/ex_aws/auth.ex

```diff
@@ -207,6 +209,7 @@ defmodule ExAws.Auth do
   defp signed_headers(headers) do
     headers
     |> Enum.map(fn {k, _} -> String.downcase(k) end)
+    |> Kernel.--(@unsignable_headers)
```

Comment:
Because of the downcase in the previous line you shouldn't need multiple cases.

Reply:
👍 Hopefully I've addressed this now, sorry about the delay

Reply:
@benwilson512 Would you possibly be able to review? Thanks